lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -465,6 +465,13 @@ function dosomething_user_new_user($form, &$form_state) {
   global $user;
   $account = $user;
 
+  if (module_exists('dosomething_global') {
```

Comment:
_GET OFF MY CODE!_ :shakes cane:

yeah ill bump it up. after that good to merge?